R/retrieveData.R

```diff
@@ -190,6 +194,7 @@ retrieveData <- function(model, rev = 0, dev = "", cachetype = "rev", puc = iden
             file.copy(cacheFiles, ".")
             otherFiles <- c("config.rds", "diagnostics.log", "diagnostics_full.log")
             file.copy(file.path(sourcefolder, otherFiles), ".")
+            trash <- capture.output(snapshot(packages = attr(cfg$functionName, "package"), prompt = FALSE)) # nolint
```

Comment:
Maybe this info could be helpful for debugging, so maybe `vcat(3, ...)`?

Reply:
good  suggestion! done.